app/code/Magento/ConfigurableProductGraphQl/Model/Resolver/Variant/Attributes/ConfigurableAttributeIdV2.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ConfigurableProductGraphQl\Model\Resolver\Variant\Attributes;
+
+use Magento\Eav\Model\ResourceModel\Entity\Attribute;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Format new option id_v2 in base64 encode for super attribute options
+ */
+class ConfigurableAttributeIdV2 implements ResolverInterface
+{
+    /**
+     * Option type name
+     */
+    private const OPTION_TYPE = 'configurable';
+
+    /**
+     * @var Attribute
+     */
+    private $eavAttribute;
+
+    /**
+     * @param Attribute $eavAttribute
+     */
+    public function __construct(Attribute $eavAttribute)
+    {
+        $this->eavAttribute = $eavAttribute;
+    }
+
+    /**
+     * Create a option id_v2 for super attribute in "<option-type>/<attribute-id>/<value-index>" format
+     *
+     * @param Field $field
+     * @param ContextInterface $context
+     * @param ResolveInfo $info
+     * @param array|null $value
+     * @param array|null $args
+     *
+     * @return string
+     *
+     * @throws GraphQlInputException
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (empty($attribute_id)) {
```

Comment:
I believe `$attribute_id` is supposed to be a part of the `$value` array but not a standalone variable